packages/ethers/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # @web3modal/ethers
 
+## 4.1.6
+
+### Patch Changes
+
+- Fix modal default height. Fix ethers email connection lag. Fix ethers + rc relay disconnect issue. Adds new wui-components for secure site.
```

Comment:
Maybe not for this release but for the others, can we have the same changelog changes list as the description of this PR? It's a much better format. 

I guess we could achieve the same by using some custom Changelog template. It's a big headache to setup but looks much cleaner.

Reply:
Was just looking into it (I feel the same way). 
I think we could use the `changelog` property in `.changesets/config.json` but we'd need to provide a custom formatting function for it. I think it's still in-dev 😵  
https://github.com/changesets/changesets/blob/main/docs/modifying-changelog-format.md